src/js/queries/servicePageFragment.js

```diff
@@ -0,0 +1,65 @@
+const servicePageFragment = `
+  fragment servicePageInfo on ServicePageNode {
+    id
+    title
+    slug
+    topic {
+      id
+      slug
+      text
+      theme {
+        id
+        slug
+        text
+      }
+    }
+    serviceSteps {
```

Comment:
From the looks of our inbound public health content, it seems like we need a title field and an unknown length of steps (which could include nested lists) here.

Reply:
I noticed the need for a title [when updating the yaml in joplin]((https://github.com/cityofaustin/joplin/pull/155/files#diff-9c5ea5128d161862f6213b2beb0cf3f2R12). Could you clarify what you mean by an unknown length of steps?